File: src/main/java/gg/mineral/practice/util/messages/ListElementMessage.java
package gg.mineral.practice.util.messages;

public class ListElementMessage extends ChatMessage {

    public ListElementMessage(String s, String c) {
        super(s, c);
    }

    @Override
    protected void formatMessage(String c, boolean bold) {
        this.prefix = CC.ACCENT + "✱ " + c;
        this.messageBuilder.insert(0, prefix);
        this.messageBuilder.append(CC.SECONDARY);
    }
}

File: src/main/kotlin/gg/mineral/practice/util/messages/ListElementMessage.kt
package gg.mineral.practice.util.messages

class ListElementMessage(s: String, c: String) : ChatMessage(s, c) {
    override val prefix by lazy {
        CC.ACCENT + "✱ " + colorPrefix
    }

    override fun prepend(builder: StringBuilder): StringBuilder = super.prepend(builder).append(CC.SECONDARY)
}
